### ISSUE TITLE
[less] syntax/linter issues

### ISSUE THREAD

**Issue by valeriiRadko** [microsoft/vscode#68324]
Issue Type: <b>Bug</b>

Next code alert following problems for the .class2 block:
1. } expected
2. at-rule or selector expected
3. at-rule or selector expected

Less code:
```
.screen(@rules) {
  @media screen and (max-width: 1200px) { @rules(); }
}
.class1 {
  .screen({font-size: 48px;});
}
.class2  {
  font-size: 100px;
  .screen ({font-size: 48px;});
}
```

According to documentation of less it should be valid syntax.

VS Code version: Code 1.31.0 (7c66f58312b48ed8ca4e387ebd9ffe9605332caa, 2019-02-05T22:32:14.164Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz (4 x 2300)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 3|
|Memory (System)|16.00GB (5.99GB free)|
|Process Argv|-psn_0_118813|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
All-Autocomplete|Ati|0.0.18
vscode-eslint|dba|1.8.0
gitlens|eam|9.5.0
sftp|lix|1.9.3
vscode-docker|Pet|0.5.2
trailing-spaces|sha|0.2.11
indenticator|Sir|0.6.0
vscode-w3cvalidation|Umo|2.3.0

</details>

**Comment by octref** [comment]
Is this less code?

**Comment by octref** [comment]
Upstream issue: https://github.com/microsoft/vscode-css-languageservice/issues/186

First two cases already work, it's only reporting error when using parametric mixins with extra space before `(`.

**Comment by aeschli** [comment]
Fixed by microsoft/vscode-css-languageservice#186
